src/WebApp/Startup.cs

```diff
@@ -124,5 +133,137 @@ private void ApplyDatabaseMigrations(IApplicationBuilder app, ILogger logger)
                 }
             }
         }
+
+        private void SeedDatabase(IApplicationBuilder app, ILogger logger)
```

Comment:
DB seed should probably be split off into a separate file/task at some point. Not one for this PR, though.